media-libs/embree/embree-4.3.1.ebuild

```diff
@@ -29,10 +29,6 @@ DEPEND="${RDEPEND}"
 
 DOCS=( CHANGELOG.md README.md readme.pdf )
 
-PATCHES=(
-	"${FILESDIR}"/embree-4.3.0-dont-install-tutorials.patch
```

Comment:
Why are you removing this patch?
Without this we will install tutorial binaries on the system that are only useful for the tests.